build.gradle.kts

```diff
@@ -90,6 +100,31 @@ publishing {
     }
 }
 
+subprojects {
+    dependencyLocking {
+        lockAllConfigurations()
+    }
+}
+
+tasks.register("createClasspathManifest") {
```

Comment:
What is this task for? I found it in [Gradle docs](https://docs.gradle.org/current/userguide/test_kit.html) saying it's needed for very old versions of Gradle (before 2.13).

Reply:
I wasn't able to get automatic class path working. Might be a problem with the way how the root build project is configured.